src/types/config.ts

```diff
@@ -3,8 +3,9 @@ interface RelatedConfig {
   autoContinueTime: number;
   showOnPlaybackDone: boolean;
   showOnPlaybackPaused: boolean;
-  playlistId?: string;
+  playlistId: string | null;
   entryList?: Array<string>;
```

Comment:
entryList should be Array of [ProviderEntryListObject](https://github.com/kaltura/playkit-js-providers/blob/master/flow-typed/types/entry-list.js) like [loadPlaylistByEntryList](https://github.com/kaltura/kaltura-player-js/blob/master/src/kaltura-player.js#L219) gets (to be supported for OTT BE as well)

Reply:
entry id is not enough for OTT ?

Reply:
no. and also need to support ks

Reply:
array of any says nothing to an app developer wants to know what the plugin gets in the config

Reply:
ok, fair enough.

Reply:
updated the types